### DISCUSSION TITLE
How to troubleshoot ACL errors when Obtain OAuth token using SN Utils?

### DISCUSSION THREAD

**Discussion by niudaye123** [arnoudkooi/SN-Utils#528] [Q&A]
Hi, 
I am trying to troubleshoot API level ACL Validation error when trying to obtain a token from an OAuth Provider, Ansible Tower. My service account is able to sign in to Provider and confirm access, but getting the ACL error. Is there an easy way to troubleshoot this with SN Utils?

**Comment by arnoudkooi** [comment]
not really, I think it's a matter of searching system logs and node logs and so on for details.
